crates/contracts/src/bin/vendor.rs

```diff
@@ -48,7 +48,7 @@ fn run() -> Result<()> {
         )?
         .github(
             "BalancerV2StablePoolFactory",
-            "balancer-labs/balancer-subgraph-v2/2b97edd5e65aed06718ce64a69111ccdabccf048/\
+            "balancer-labs/balancer-v2-monorepo/ad1442113b26ec22081c2047e2ec95355a7f12ba/\
```

Comment:
Unrelated change, but this just fixes an issue where the `vendor` script was failing because the URI to this artifact was wrong (accidentally introduced in https://github.com/gnosis/gp-v2-services/pull/1403).